collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -629,7 +630,10 @@ private boolean loadSharedPreferencesFromFile(File src) {
             // first object is preferences
             Map<String, ?> entries = (Map<String, ?>) input.readObject();
 
-            migrateAutosendPrefKeys(entries, prefEdit);
+            if (entries.containsKey(PreferenceKeys.KEY_AUTOSEND_WIFI)
```

Comment:
I don't believe you need these tests here. You can simply call `migrate` and the checks will be done.